crates/eframe/src/native/run.rs

```diff
@@ -211,30 +210,43 @@ fn run_and_return(
     // we only apply this approach on Windows to minimize the affect.
     #[cfg(target_os = "windows")]
     {
-        event_loop.run_return(|_, _, control_flow| {
+        event_loop.run_on_demand(|_, _, control_flow| {
             control_flow.set_exit();
         });
     }
```

Comment:
Can someone one Windows check if this hack is still needed, or if `cargo run -p serial_windows` works without it?